src/EnergyPlus/TranspiredCollector.cc

```diff
@@ -1172,14 +1188,14 @@ namespace TranspiredCollector {
 		UTSC( UTSCNum ).Tcoll = TmpTscoll;
 		UTSC( UTSCNum ).HrPlen = HrPlen;
 		UTSC( UTSCNum ).HcPlen = HcPlen;
-		UTSC( UTSCNum ).TairHX = 0.0;
+		UTSC( UTSCNum ).TairHX = Tamb;
 		UTSC( UTSCNum ).InletMDot = 0.0;
 		UTSC( UTSCNum ).InletTempDB = Tamb;
 		UTSC( UTSCNum ).Vsuction = 0.0;
 		UTSC( UTSCNum ).PlenumVelocity = 0.0;
-		UTSC( UTSCNum ).SupOutTemp = Tamb;
+		UTSC( UTSCNum ).SupOutTemp = TmpTaPlen;
 		UTSC( UTSCNum ).SupOutHumRat = OutHumRatAmb;
-		UTSC( UTSCNum ).SupOutEnth = OutEnthalpy;
+		UTSC( UTSCNum ).SupOutEnth = PsyHFnTdbW( Tamb, OutHumRatAmb );
```

Comment:
I think this needs to be TmpTaPlen now instead of Tamb?  And is OutHumRatAmb appropriate?  I suppose that makes sense here.